tensorflow/compiler/xla/service/gpu/gpu_device_info_for_tests.cc

```diff
@@ -53,8 +53,8 @@ GpuDeviceInfo TestGpuDeviceInfo::AMDMI210DeviceInfo() {
   info.core_count = 104;
   info.fpus_per_core = 0;
   info.block_dim_limit_x = 2'147'483'647;
-  info.block_dim_limit_y = 2'147'483'647;
-  info.block_dim_limit_z = 2'147'483'647;
+  info.block_dim_limit_y = 65536;
```

Comment:
is only rocm6.0 has this output? I tried rocm-5.4 on my two nodes, and results are different to yours https://github.com/ROCmSoftwarePlatform/frameworks-internal/issues/6124#issuecomment-1778275074 . I'm really doubt this y and z results so different to x, it doesn't make sense.

Reply:
Hmm, if it shows different results for different rocm versions, we will need to file a JIRA ticket then.